utils/manifests/getComponents.go

```diff
@@ -0,0 +1,38 @@
+package manifests
+
+import (
+	"github.com/layer5io/meshkit/utils"
+	k8s "github.com/layer5io/meshkit/utils/kubernetes"
```

Comment:
Why there included the binary file `utils/manifests/utilBin/kubeopenapi-jsonschema` , `utils/manifests/utilBin/kubeopenapi-jsonschema-darwin` ?

Reply:
+1. Ideally, we are not including these binaries local to the MeshKit package. ￼

Reply:
@leecalcote  you told me last time to package them with binary, 
do you prefer to fetch them on the fly?

Reply:
I think we would be fetching the binary for jsonschema utility on demand. (when the user demands to generate components).
Shipping it in either this library or the adapter image is not needed. We don't expect users to grab new service mesh manifests and generate components in air gapped deployments.